### PR TITLE
Macros: sidebars and links

### PR DESCRIPTION
**Pull request by estelle**
Follow up to https://github.com/mdn/content/pull/31248

Note re: create a sidebar section::

I removed the "how to create a sidebar" as people should not be creating one-off sidebars. I changed the lingo to "temporarily include a new sidebar". I think it's helpful to have it documented so that main contributors have something to look at and play with as they develop new sections of content. i am clear that it won't be approved. But I will be playing with "quick_links" as i create the A11y section sidebar, and will keep it around until such sidebar is merged into yari.

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/sidebars/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+### Creating new sidebars
+
+You should use existing sidebar macros, without adding any content to them. If you are creating a whole new section of content, create a [macro for your sidebar](https://github.com/mdn/yari/blob/main/kumascript/macros/) in Yari.
+
+In the unlikely event that you need to create a temporary sidebar, this section explains how that can be done. Do not submit your temporary sidebar for PR review as it will not be approved.
+
+If you need to create a new sidebar macro, you can do so in your development environment by following these steps:
+
+1. Remove the sidebar sidebar macro appearing immediately after the frontmatter and before the content, as each document can only have one sidebar.
```
Comment (dawei-wang) [inline files/en-us/mdn/writing_guidelines/page_structures/sidebars/index.md:78]:
```suggestion
1. Remove the sidebar macro appearing immediately after the frontmatter and before the content, as each document can only have one sidebar.
```

File: files/en-us/mdn/writing_guidelines/page_structures/sidebars/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+In this guide, you will learn how to create a sidebar of links by just including a macro and how to create sidebars with additional content.
+
+## Creating sidebars
+
+Every page has a sidebar. These sidebars appear because a sidebar macro has been included on the pages that create a potentially hierarchical list of links to other pages on this site.
+
+When a sidebar macro is included, the server creates a section of content containing an unordered list of links. The links created, where they are displayed, and how they are displayed depend on the macro used and the parameters included in the markdown macro call. Some sidebars include links based on a directory's structure or page type. Others include a list of predefined pages that are hard-coded in Yari.
+
+### including single-macro sidebars
```
Comment (dawei-wang) [inline files/en-us/mdn/writing_guidelines/page_structures/sidebars/index.md:20]:
```suggestion
### Including single-macro sidebars
```

**Review by dawei-wang — COMMENTED** [review]
Delete repeated word. Capitalize heading.

File: files/en-us/mdn/writing_guidelines/page_structures/links/index.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  - : Inserts a definition list ({{HTMLelement("dl")}}) of the subpages of the current page, with each page's title as the {{HTMLelement("dt")}} term and its SEO summary as the {{HTMLelement("dd")}} term. The summary is the first paragraph of content.
+
+- [`\{{ListSubpagesForSidebar()}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/ListSubpagesForSidebar.ejs)
+
+  - : When included without parameters, inserts an ordered list of links to the current page's subpages. This macro is most often used within [sidebars](/en-US/docs/MDN/Writing_guidelines/Page_structures/Sidebars#sidebars_adding_additional_content) (hence the macro name), where the bullets are not rendered. The first parameter is a slug of the link tree's parent page. The link text is displayed as code. Setting a second parameter to `true` or `1` converts the links to plain text. Setting a third parameter to `true` or `1` adds a link the slug (parent) page at the top of the list with "Overview" as the link text.
+
+  - [`\{{QuickLinksWithSubpages`()}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/QuickLinksWithSubpages.ejs)
```
Comment (estelle) [inline files/en-us/mdn/writing_guidelines/page_structures/links/index.md:23]:
```suggestion
- [`\{{QuickLinksWithSubpages`()}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/QuickLinksWithSubpages.ejs)
```

File: files/en-us/mdn/writing_guidelines/page_structures/links/index.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  - : Inserts a definition list ({{HTMLelement("dl")}}) of the subpages of the current page, with each page's title as the {{HTMLelement("dt")}} term and its SEO summary as the {{HTMLelement("dd")}} term. The summary is the first paragraph of content.
+
+- [`\{{ListSubpagesForSidebar()}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/ListSubpagesForSidebar.ejs)
+
+  - : When included without parameters, inserts an ordered list of links to the current page's subpages. This macro is most often used within [sidebars](/en-US/docs/MDN/Writing_guidelines/Page_structures/Sidebars#sidebars_adding_additional_content) (hence the macro name), where the bullets are not rendered. The first parameter is a slug of the link tree's parent page. The link text is displayed as code. Setting a second parameter to `true` or `1` converts the links to plain text. Setting a third parameter to `true` or `1` adds a link the slug (parent) page at the top of the list with "Overview" as the link text.
+
+- [`\{{QuickLinksWithSubpages`()}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/QuickLinksWithSubpages.ejs)
```
Comment (estelle) [inline files/en-us/mdn/writing_guidelines/page_structures/links/index.md:23]:
```suggestion
- [`\{{QuickLinksWithSubpages()}}`](https://github.com/mdn/yari/blob/main/kumascript/macros/QuickLinksWithSubpages.ejs)
```

**Pull request closed without merging**